src/DtlsSocket.cc

```diff
@@ -158,13 +173,18 @@ DtlsSocket::DtlsSocket(const unsigned char *priv_key,
 																		MBEDTLS_SSL_TRANSPORT_DATAGRAM,
 																		MBEDTLS_SSL_PRESET_DEFAULT);
 	if (ret != 0) goto exit;
-	mbedtls_ssl_conf_min_version(&conf, MBEDTLS_SSL_MAJOR_VERSION_3, MBEDTLS_SSL_MINOR_VERSION_3);
+	mbedtls_ssl_conf_min_version(&conf, MBEDTLS_SSL_MAJOR_VERSION_3, MBEDTLS_SSL_MINOR_VERSION_2);
```

Comment:
why do we use version 2 instead of 3?

Reply:
Same reason as the for node-mbed-dtls.  Z/IP Gateway requires support for TLS v1.2.